src/main/java/fm/last/moji/local/LocalMojiFileAttributes.java

```diff
@@ -48,4 +48,9 @@ public String getKey() {
     return mojiFile.getKey();
   }
 
+  @Override
+  public String getChecksum() {
+    return "";
```

Comment:
I thought this should work like getFid(), in that it fulfills the contract of the MojiFileAttributes interface, but that a LocalMojiFile doesn't have a fid, or checksum. I have changed this to return null rather than an empty string. If my understanding of these LocalMojiFile classes is wrong, I'd be happy to make additional changes.